src/ripple/protocol/impl/tokens.cpp

```diff
@@ -77,12 +88,16 @@ digest2 (Args const&... args)
     @note This checksum algorithm is part of the client API
 */
 void
-checksum (void* out,
-    void const* message,
-        std::size_t size)
+checksum(void* out, void const* message, std::size_t size)
+{
+    auto const h = digest2<sha256_hasher>(message, size);
+    std::memcpy(out, h.data(), 4);
```

Comment:
Where does 4 come from? -> put in a constexpr?

Reply:
Addressed in 91c6c712b0973af30980b849d50e33a4f715d449